website/src/content/main-concepts/user-permissions.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: User Permissions
+---
+
+<Info>
+
+See [User Permission Concepts](https://docs.commercetools.com/merchant-center/user-permissions.html).
+
+</Info>
+
+_Coming soon_
```

Comment:
Ref #1264

Reply:
@tdeekens is going to write a draft soon, so I'll like to keep the page since it's also linked from other pages.